app/assets/javascripts/discourse/app/widgets/do-not-disturb.js

```diff
@@ -11,21 +11,10 @@ export default createWidget("do-not-disturb", {
   saving: false,
 
   html() {
-    if (this.currentUser.isInDoNotDisturb()) {
-      return [
-        h("button.btn-flat.do-not-disturb-inner-container", [
-          iconNode("toggle-on"),
-          this.label(),
-        ]),
-      ];
-    } else {
-      return [
-        h("button.btn-flat.do-not-disturb-inner-container", [
-          iconNode("toggle-off"),
-          h("span.do-not-disturb-label", I18n.t("pause_notifications.label")),
-        ]),
-      ];
-    }
+    const isOn = this.currentUser.isInDoNotDisturb();
+    const button = this._menuButton(isOn);
+
+    return [button];
```

Comment:
Just a personal preference, but this function looks longer than it should be.

```suggestion
    return [this._menuButton(isOn)];
```